### DISCUSSION TITLE
Some peers (or trackers) are likely blocking the name "Transmission"

### DISCUSSION THREAD

**Discussion by Coeur** [transmission/transmission#4582] [General]
I had some downloads stuck at 0% for one month (yeah, I'm patient for rare downloads).
And today, I randomly renamed my fork, just that (a renaming of TR_NAME), and a few seconds after, those downloads that had been stuck for an eternity started downloading from a peer.

It can't be a coincidence: I'm restarting Transmission multiple times a day since I rebuild each time there are changes on the main branch, but the only time that those very old torrents started downloading is when I renamed my client to something different than "Transmission".

**Comment by Pentaphon** [comment]
But why? It's a well-regarded client.

**Comment by ckerr** [comment]
This is a pretty big claim and it would be pretty hard to add a client blocker to an open-source project.

So it's unlikely but _possible_ for uTorrent to be doing this since it's closed source, but if e.g. qBitTorrent or Deluge etc. were doing this then we could go into the source code to confirm it. If you think you're seeing this for particular open-source clients, the next step would be for you to go check out the source and confirm.

Tangentally related to this, I _do_ think that Transmission _can_ connect to fewer peers in a swarm since it doesn't support the holepunch bep (that's milestoned for Transmission 5).

**Reply by Coeur** [reply]
Ah, I didn't think about looking at the peer client. I'll check it next time I notice. Presently, my peers are "utorrent", but doesn't mean they are the same as yesterday.

**Comment by cristiandan** [comment]
silly question, but is there a way to change the reported UA/string? (eg. Transmit 4.0.0-beta.1) without forking and rebuilding

**Reply by ckerr** [reply]
It requires editing `version.h` and rebuilding

**Reply by Pentaphon** [reply]
> silly question, but is there a way to change the reported UA/string? (eg. Transmit 4.0.0-beta.1) without forking and rebuilding

Not worth the trouble.
